### PR TITLE
Add Private Internet Access

### PR DESCRIPTION
**Pull request by leetruitt**

### CODE REVIEW

**Review by stephengroat — APPROVED** [review]
thanks for the pr @leetruitt!

File: _data/security.yml [hunk 1]
```diff
@@ -124,6 +124,12 @@ websites:
       software: Yes
       doc: http://help.preyproject.com/article/248-how-to-enable-the-two-step-authenticator-for-prey
 
+    - name: Private Internet Access
+      url: https://www.privateinternetaccess.com
+      img: privateinternetaccess.png
+      tfa: No
+      twitter: buyvpnservice
```
Comment (Carlgo11) [inline _data/security.yml]: Please change their twitter handle to `piacsm`

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/security.yml (hunk 1 above)
Comment (disforw) [reply _data/security.yml]: The doc is wrong ??

Comment (Carlgo11) [reply _data/security.yml]: No but the twitter handle is wrong

Comment (leetruitt) [reply _data/security.yml]: Sorry about that. The twitter handle should be correct now.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**